locksmith/src/graphql/datasource/keysByQuery.ts

```diff
@@ -159,6 +159,8 @@ export class keysByQuery extends GraphQLDataSource {
 
       const getData = async (getFromPage = page) => {
         const skip = parseInt(`${getFromPage}`, 10) * first
+        // The Graph does not support skipping more than 5000
+        // https://thegraph.com/docs/en/querying/graphql-api/#pagination
```

Comment:
So this will fail, but that's ok because now we handle the failure in a way that allows us to return partial results.